charts/steward/README.md

```diff
@@ -109,7 +109,7 @@ Pipeline Run Controller:
 | <code>runController.<wbr/>tolerations</code> | (array of [`Toleration`][k8s-tolerations])<br/> The `tolerations` field of the Run Controller [pod spec][k8s-podspec]. | `[]` |
 | <code>runController.<wbr/>args.<wbr/>qps</code> | (integer)<br/> The maximum queries per second (QPS) from the controller to the cluster. | 5 |
 | <code>runController.<wbr/>args.<wbr/>burst</code> | (integer)<br/> The burst limit for throttle connections (maximum number of concurrent requests). | 10 |
-| <code>runController.<wbr/>args.<wbr/>logVerbosity</code> | (integer)<br/> The log verbosity. 0=error, 1=warning, 2=info, 3=extended, 4=debug, 5=trace (see also [logging-conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#logging-conventions)) | 3 |
+| <code>runController.<wbr/>args.<wbr/>logVerbosity</code> | (integer)<br/> The log verbosity. 0=error, 1=warning, 2=info, 3=extended, 4=debug, 5=trace (see also [logging-conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#logging-conventions)) | 2 |
```

Comment:
The level description is not appropriate.

Reply:
Can you be more specific? We could also remove the description and only refer to the kubernetes logging conventions.
```suggestion
| <code>runController.<wbr/>args.<wbr/>logVerbosity</code> | (integer)<br/> The log verbosity. Levels adopted from [kubernetes logging conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#logging-conventions) | 2 |
```

Reply:
0-level logs are not always errors, 1-level logs are not always warnings and so on. Actually they are just relative levels.

Lena provided a better-suited level description [above](https://github.com/SAP/stewardci-core/pull/147#issuecomment-689353224). In the chart documentation I would put this into a separate section (not include it in the description of individual parameters).

Reply:
The level description of the K8s logging convention are also fine.

Reply:
The [k8s logging conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#logging-conventions) differ from Lenas original description [above](https://github.com/SAP/stewardci-core/pull/147#issuecomment-689353224). We agreed it would make sense to stick to the k8s best practices.

Therefore I would recommend to simply link to it (as proposed in my change.

Reply:
I go with Alex's approach. Its better to stick with the logging conventions. Also because the packages we use, might use klog and this conventions, too.